clearpath_config/manipulators/types/arms.py

```diff
@@ -27,10 +27,12 @@
 # POSSIBILITY OF SUCH DAMAGE.
 from typing import List
 from clearpath_config.common.types.accessory import Accessory
+from clearpath_config.common.types.exception import UnsupportedAccessoryException
 from clearpath_config.common.types.ip import IP
 from clearpath_config.common.types.port import Port
 from clearpath_config.manipulators.types.grippers import Gripper
 from clearpath_config.manipulators.types.manipulator import BaseManipulator
+from clearpath_generator_common.ros import ROS_DISTRO
```

Comment:
Can `ROS_DISTRO` be change to `RosDistro` to match everything else and avoid confusion?

Reply:
That would require changing https://github.com/clearpathrobotics/clearpath_common/blob/jazzy-2.0RC/clearpath_generator_common/clearpath_generator_common/ros.py, which we can do, I suppose.

Personally I don't like this proposal though, since `RosDistro` looks like it's a class, but `ROS_DISTRO` is clearly intended to be a constant (and matches the well-known envar).

Reply:
Turns out it doesn't actually matter; we can't use `clearpath_generator_common` since it depends on `clearpath_common`. Otherwise we wind up with circular dependencies.

I've refactored this package to just look up the envar, defaulting to `jazzy` if it's not found for some arcane reason.